packages/hop-node/src/watchers/xDomainMessageRelayWatcher.ts

```diff
@@ -19,12 +23,20 @@ type Config = {
   dryMode?: boolean
 }
 
+type ConfirmRootData = {
```

Comment:
This type can be exported so it's used in cli file

```suggestion
export type ConfirmRootData = {
```